lib/operator.rb

```diff
@@ -0,0 +1,78 @@
+require_relative 'menu.rb'
+require_relative 'text.rb'
+
+class Operator
+
+  def initialize(menu = Menu.new, text = Text.new)
+    @menu = menu
+    @text = text
+    @current_text = nil
+  end
+
+  def check_new_messages
+    @current_text = @text.new_message
+  end
+
+  def action
+    raise 'You have not checked the inbox' if @current_text == nil
```

Comment:
Style/NilComparison: Prefer the use of the `nil?` predicate. (https://github.com/bbatsov/ruby-style-guide#predicate-methods)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)